src/BUILD.gn

```diff
@@ -92,9 +92,9 @@ if (chip_build_tests) {
       deps += [ "${chip_root}/src/ble/tests" ]
     }
 
-    if (!is_clang && chip_device_platform != "darwin" &&
-        chip_device_platform != "nrfconnect" &&
-        chip_device_platform != "efr32") {
+    # For some reason linking the controller tests fails on nrfconnect with
+    # complaints about multiple definitions of various functions.
```

Comment:
Is it this one? https://github.com/project-chip/connectedhomeip/issues/9630

It's a real issue and I believe it would always happen if linking libCHIP or libCHIP_tests with whole-archive.

Reply:
@mspang Yes, that is exactly the link error I got.  I'll add a reference to that issue.